openfifth/aspen-discovery
# 24.07.00 grapes js web builder integration

Test Plan: 
1. Enable the Web Builder Module by navigating to System Administration -> Modules -> WebBuilder. 
2. Once enabled, notice that there are now two new options in Web Builder: Grapes Pages and Templates.
3.  Click on templates and add new. Give your new template a name and save. 
4.  Click 'Open in Editor' to open your template in the Grapes Editor. 
5. Add some content using the grapes editor. 
6. Click the save button and then navigate back to your Web Builder Section. 
7. Click on Grapes Pages and add a new page. 
8. Fill in your page details and select a template. Click save. 
9. Use the 'Open in Editor' button to open the page in the editor. Note it contains the html information form your template. 
10. Add to your page and click save. 
11. Reload the page and note that your new page data is persistent. 
12. Navigate to your page using the URL you gave in the Grapes Page set up.
